modules/mobile/spec/helpers/mobile/appointments_cache_interface_spec.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
```

Comment:
These tests may seem like a mess of stubbing, but I wanted them to be "pure" unit tests. I think we've become overly reliant on integration specs, which is leading to bloated request specs in which we essentially retest the entire suite "with flag on/off" even when the flag has no impact on the tested behavior. That's tedious and unsustainable. I'd rather the tedium be in unit tests so that we don't end up with 1k+ line request specs that are unreadable and difficult to add to and maintain.